com.unity.formats.fbx/Tests/FbxApiTests/SampleCodeTest.cs

```diff
@@ -0,0 +1,142 @@
+using UnityEngine;
+using UnityEditor;
+using NUnit.Framework;
+using System.Linq;
+using UnityEditor.Formats.Fbx.Exporter;
+using System.IO;
+using Autodesk.Fbx;
+
+namespace FbxExporter.UnitTests
+{
+    /// <summary>
+    /// Unit tests for sample code included in documentation.
+    /// </summary>
+    public class SampleCodeTest
+    {
+        private const string k_tempFolderName = "_safe_to_delete";
+        private string m_tempFolder = Path.Combine(Application.dataPath, k_tempFolderName);
+
+        [SetUp]
+        public void Init()
+        {
+            Assert.That(m_tempFolder, Does.Not.Exist);
+            Directory.CreateDirectory(m_tempFolder);
+        }
+
+        [TearDown]
+        public void Term()
+        {
+            if (Directory.Exists(m_tempFolder))
+            {
+                Directory.Delete(m_tempFolder, recursive: true);
+            }
+        }
+
+        // Export GameObjects sample function
+        public static void ExportGameObjects(Object[] objects)
+        {
+            string filePath = Path.Combine(Application.dataPath, "MyGame.fbx");
```

Comment:
It would be better to use a temp folder for this, but I copied this function exactly from the docs and thought it would be best to leave as is for most accurate testing.